spyderlib/config.py

```diff
@@ -528,7 +528,6 @@ def is_ubuntu():
               'editor/open file': "Ctrl+O",
               'editor/save file': "Ctrl+S",
               'editor/save all': "Ctrl+Shift+S",
-              'editor/print': "Ctrl+P",
               'editor/close all': "Ctrl+Shift+W",

```

Comment:
Did we agree on removing print completely @ccordoba12?, I thought we were re patching to something like `` `Ctrl+Shift+P`